### PR TITLE
OBSDOCS-642: Troubleshooting logging docs cleanup

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/OBSDOCS-642

Link to docs preview:
- No landing page, but basically all the items moved to the section: https://70681--ocpdocs-pr.netlify.app/openshift-enterprise/latest/logging/troubleshooting/cluster-logging-cluster-status
- https://70681--ocpdocs-pr.netlify.app/openshift-enterprise/latest/support/#support-overview-troubleshooting-issues

QE review:
- N/A

### CODE REVIEW

File: _topic_maps/_topic_map.yml [hunk 1]
```diff
@@ -2560,6 +2560,17 @@ Topics:
     File: logging-5-7-release-notes
 - Name: Support
   File: cluster-logging-support
+- Name: Troubleshooting logging
```
Comment (mletalie) [inline _topic_maps/_topic_map.yml:2563]:
```suggestion
- Name: Troubleshooting Logging
```

**Review by mletalie — COMMENTED** [review]
/label peer-review-in-progress

File: _topic_maps/_topic_map.yml [hunk 2]
```diff
@@ -2560,6 +2560,17 @@ Topics:
...
+  Dir: troubleshooting
+  Topics:
+  - Name: Viewing Logging status
+    File: cluster-logging-cluster-status
+  - Name: Troubleshooting log forwarding
+    File: log-forwarding-troubleshooting
+  - Name: Troubleshooting logging alerts
```
Comment (mletalie) [inline _topic_maps/_topic_map.yml:2570]:
```suggestion
  - Name: Troubleshooting Logging alerts
```

File: support/index.adoc [hunk 3]
```diff
@@ -125,11 +125,11 @@ endif::openshift-rosa,openshift-dedicated[]
...
-** xref:../logging/troubleshooting/cluster-logging-cluster-status.adoc#cluster-logging-cluster-status[Status of the Log store].
-** xref:../logging/logging_alerts/troubleshooting-logging-alerts.adoc#troubleshooting-logging-alerts[Troubleshooting logging alerts].
-** xref:../logging/cluster-logging-support.adoc#cluster-logging-support-must-gather_cluster-logging-support[Information about your OpenShift logging environment using `oc adm must-gather` command].
+** xref:../logging/troubleshooting/cluster-logging-cluster-status.adoc#cluster-logging-clo-status_cluster-logging-cluster-status[Viewing the status of the {clo}].
+** xref:../logging/troubleshooting/cluster-logging-cluster-status.adoc#cluster-logging-clo-status-comp_cluster-logging-cluster-status[Viewing the status of {logging} components].
+** xref:../logging/troubleshooting/troubleshooting-logging-alerts.adoc#troubleshooting-logging-alerts[Troubleshooting logging alerts].
+** xref:../logging/cluster-logging-support.adoc#cluster-logging-support-must-gather_cluster-logging-support[Information about your logging environment using `oc adm must-gather` command].
```
Comment (mletalie) [inline support/index.adoc:133]: Wonder if this link could be/should directed to this section of the doc given the subject matter? https://70681--ocpdocs-pr.netlify.app/openshift-enterprise/latest/logging/cluster-logging-support#cluster-logging-must-gather-collecting_cluster-logging-support?

**Review by mletalie — COMMENTED** [review]
/label peer-review-done
/remove-label peer-review-needed
/remove-label peer-review-in-progress
/lgtm

**Comment by mletalie** [comment]
Hello @abrennan89,
    Just a couple of comments/otherwise lgtm. Good job!

File: _topic_maps/_topic_map.yml (hunk 2 above)
Comment (abrennan89) [reply _topic_maps/_topic_map.yml:2570]: I think Libby is working on [standardizing the naming conventions and attributes at the moment](https://issues.redhat.com/browse/OBSDOCS-481), so I probably won't make these part of the scope of this PR if that's OK.

File: _topic_maps/_topic_map.yml (hunk 1 above)
Comment (abrennan89) [reply _topic_maps/_topic_map.yml:2563]: I think Libby is working on [standardizing the naming conventions and attributes at the moment](https://issues.redhat.com/browse/OBSDOCS-481), so I probably won't make these part of the scope of this PR if that's OK.

Comment (mletalie) [reply _topic_maps/_topic_map.yml:2563]: Sure thing, thanks for the heads up!

**Pull request merged**